docs/tutorials/blog.rst

```diff
@@ -47,234 +134,280 @@ Create the file ``/views/default/resources/my_blog/add.php``.
+
+To be able to view a my_blog post on its own page, you need to make a view page.
+Create the file ``/mod/my_blog/views/default/resources/my_blog/view.php``:
+
+.. code-block:: php
+
+    <?php
+
+    // get the entity
+    $guid = elgg_extract('guid', $vars);
+    $my_blog = get_entity($guid);
+
+    // get the content of the post
+    $content = elgg_view_entity($my_blog, array('full_view' => true));
+
+    $params = array(
+        'title' => $my_blog->title,
+        'content' => $content,
+        'filter' => '',
+    );
+
+    $body = elgg_view_layout('content', $params);
+
+    echo elgg_view_page($my_blog->title, $body);
+
+This page has much in common with the ``add.php`` page. The biggest differences
+are that some information is extracted from the my_blog entity, and instead of
+viewing a form, the function ``elgg_view_entity`` is called. This function
+gives the information of the entity to something called the object view.
+
+Create the object view
+======================
+
+When ``elgg_view_entity`` is called or when my_blogs are viewed in a list
+for example, the object view will generate the appropriate content.
+Create the file ``/mod/my_blog/views/default/object/my_blog.php``:
+
+.. code-block:: php
+
+    <?php
+    
+    echo elgg_view('output/longtext', array('value' => $vars['entity']->description));
+    echo elgg_view('output/tags', array('tags' => $vars['entity']->tags)); 
+
+As you can see in the previous section, each my\_blog post is passed to the object
+view as ``$vars['entity']``. (``$vars`` is an array used in the views system to
+pass variables to a view.)
+
+The last line takes the tags on the my\_blog post and automatically
+displays them as a series of clickable links. Search is handled
+automatically.
+
+(If you're wondering about the "``default``" in ``/views/default/``,
+you can create alternative views. RSS, OpenDD, FOAF, mobile and others
+are all valid view types.)
 
 Trying it out
 =============
 
-The page to create a new my\_blog post should be accessible at ``https://elgg.example.com/my_blog/add``.
+Go to your Elgg site's administration page, list the plugins and activate
+the ``my_blog`` plugin.
 
-Displaying list of my\_blogs
-============================
+The page to create a new my\_blog post should now be accessible at
+``https://elgg.example.com/my_blog/add``, and after successfully saving the post,
+you should see it viewed on its own page.
+
+Displaying a list of blog posts
+===============================
 
 Let's also create a page that lists my\_blog entries that have been created.
 
-Create ``/views/default/resources/my_blog/all.php``.
+Create ``/mod/my_blog/views/default/resources/my_blog/all.php``:
 
-To grab the latest my\_blog posts, we'll use ``elgg_list_entities``.
-Note that this function returns only the posts that the user can see,
-so access restrictions are handled transparently:
+.. code-block:: php
 
-.. code:: php
+    <?php
+    $titlebar = "All Site My_Blogs";
```

Comment:
Ah, I thought you only referred to the three lines shown in the comment view, but it’s about the line below that has $pagetitle which should be $page_title. Check.